CMakeLists.txt

```diff
@@ -2038,6 +2047,8 @@ if(ENGINEPRIME)
     message(STATUS "Using existing system installation of libdjinterop")
     target_include_directories(mixxx-lib PUBLIC ${DjInterop_INCLUDE_DIRS})
     target_link_libraries(mixxx-lib PRIVATE DjInterop::DjInterop)
+    find_package(ZLIB 1.2.8 REQUIRED)
+    target_link_libraries(mixxx-lib PRIVATE ${ZLIB_LIBRARIES})
```

Comment:
Does this change relate to Qt6 or is it something we need in the 2.4 branch anyway?

Reply:
Qt6 only, and possibly because I am using the official Qt6 binary install. I suppose we should use a vcpkg buildenv with Qt6 instead?

Reply:
Yes. I will take a look to provide such a package.

Reply:
Btw, with the current buildenv I get this warning for all Qt libraries when linking: warning: dylib (/Users/m0dB/Qt6/6.5.0/macos/lib/QtPrintSupport.framework/Versions/A/QtPrintSupport) was built for newer macOS version (11.0) than being linked (10.15)

Reply:
You use the upstream distribution of QT, right? Since Qt 6.5 they build by default with a minimum of macOS 11.00
without a technical reason https://github.com/qt/qtbase/commit/f2415df24de6860048bd0dd4c3e3ec40f867ee53, while hey still allow to build for 10.15 
https://github.com/qt/qtbase/blob/38c8eb8564495b0fe345eaca700966889f1f8aa3/cmake/QtBuild.cmake#L235

If we we do not use 11.0 only API,  I think we should not drop 10.15 in our own build artificially.

Reply:
Oh, I just noticed that it the minimum requirement is macOS 10.14 
https://github.com/qt/qtbase/commit/4933a5f89231afd64b1604d47220e1ff34d2c448

My build confirms it: 
https://github.com/daschuer/vcpkg/actions/runs/5125452456

Reply:
I agree and it's a good reason to use Qt6 from our own buildenv.